main.py

```diff
@@ -1,22 +1,40 @@
 import os
```

Comment:
Maybe it would be nice to be able to use command line arguments so that we don't have to modify the script each time we want to use it

Reply:
True, but this is only used for testing locally.